src/Android/Effects/NoEmojiKeyboardEffect.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Bit.Droid.Effects;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ExportEffect(typeof(NoEmojiKeyboardEffect), "NoEmojiKeyboardEffect")]
```

Comment:
Use `nameof(NoEmojiKeyboardEffect)`, in case the name of the effect changes the refactor will be easier